cloudbuild.yaml

```diff
@@ -282,28 +380,31 @@ steps:
       - |
         set -e
 
+        # Switch to correct context 
+        kubectl config use-context gke_${PROJECT_ID}_${_REGION}_rag-ml-${SHORT_SHA}-${_PR_NUMBER}-${_BUILD_ID}-cluster
+
         cd /workspace/applications/rag/
         terraform destroy \
         -var-file=workloads.tfvars \
-        -var=network_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-$_AUTOPILOT_CLUSTER  \
+        -var=network_name=rag-ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-$_AUTOPILOT_CLUSTER  \
         -var=create_cluster=false \
         -var=jupyter_add_auth=false \
         -var=frontend_add_auth=false \
         -var=project_id=$PROJECT_ID \
-        -var=cluster_name=ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
+        -var=cluster_name=rag-ml-$SHORT_SHA-$_PR_NUMBER-$_BUILD_ID-cluster \
         -var=cluster_location=$_REGION \
         -var=kubernetes_namespace=rag-$SHORT_SHA-$_BUILD_ID \
         -var=gcs_bucket=gke-aieco-rag-$SHORT_SHA-$_BUILD_ID \
-        -var=ray_service_account=ray-sa-$SHORT_SHA-$_BUILD_ID \
+        -var=ray_service_account=rag-ray-sa-$SHORT_SHA-$_BUILD_ID \
         -var=rag_service_account=rag-sa-$SHORT_SHA-$_BUILD_ID \
-        -var=jupyter_service_account=jupyter-sa-$SHORT_SHA-$_BUILD_ID \
+        -var=jupyter_service_account=rag-jupyter-sa-$SHORT_SHA-$_BUILD_ID \
         -var=cloudsql_instance=pgvector-instance-$SHORT_SHA-$_BUILD_ID \
         -auto-approve -no-color
 
     allowFailure: true
     waitFor: ['test rag']
 
-  - id: 'cleanup gke cluster'
+  - id: 'cleanup jupyter jupyter ray gke cluster'
```

Comment:
I saw the cloudbuild failed at this step as well.
- Is it expected? looking at logs we didn't do anything here. 
```
Already have image (with digest): gcr.io/gke-ai-eco-dev/terraform
Switched to context "gke_gke-ai-eco-dev_us-east4_ml-c5d5cec-581-140c7add-cluster".
Switched to workspace "jupyterray".

No changes. No objects need to be destroyed.

Either you have not created any objects yet or the existing objects were
already deleted outside of Terraform.

Destroy complete! Resources: 0 destroyed.
Workspace "jupterray" doesn't exist.

You can create this workspace with the "new" subcommand 
``` 

Also the naming `jupyter jupyter`